src/target/source/codegen_opencl.cc

```diff
@@ -585,6 +586,14 @@ void CodeGenOpenCL::SetTextureScope(
 }
 
 runtime::Module BuildOpenCL(IRModule mod, Target target) {
+#if TVM_USE_VULKAN
```

Comment:
Probably this variable can be device-agnostic? Like: `TVM_USE_SPIRV` or something like that.

Reply:
Okay I'll change it to `TVM_HAS_SPIRV` which, for now, is set to true only when `USE_VULKAN=ON`.

Reply:
Changed to `TVM_ENABLE_SPIRV`, thanks for the suggestion!